app/src/msw/handlers/packetHandler.ts

```diff
@@ -8,8 +8,8 @@ export const packetHandlers = [
   rest.get(`${packetIndexUri}/metadata/${mockPacket.id}`, (req, res, ctx) => {
     return res(ctx.json(mockPacket));
   }),
-  rest.get(`${packetIndexUri}/${mockPacketGroupResponse.content[0].name}`, (req, res, ctx) => {
-    return res(ctx.json(mockPacketGroupResponse));
+  rest.get(`${packetIndexUri}/${mockPacket.name}`, (req, res, ctx) => {
+    return res(ctx.json(mockPacketGroupResponse.content));
```

Comment:
If you're returning something that depends on `mockPacketGroupResponse`, why not stick with the input depending on it too?

Reply:
that true have reverted